lessons/students/1_basics/9_indexing_and_loops.py

```diff
@@ -79,6 +83,8 @@
 
 ####################################################################################################
 
+# TITLE: Section 3.1: Using a dictionary in a loop (7 min)
+
```

Comment:
Let's make reading grades easier by storing each grade as a value to the student's name as its key in a dictionary. When you use a for loop, it will return each key in the dictionary. You can then access each value by using bracket notation: "'dictionary[key]".